### DISCUSSION TITLE
platform-pitch natural frequency of the IEA 22MW FOWT

### DISCUSSION THREAD

**Discussion by yangyangyang98** [IEAWindTask37/IEA-22-280-RWT#102] [Q&A] [answered]
I would like to inquire about the platform-pitch natural frequency of the IEA 15MW semi-submersible floating wind turbine, as well as the natural frequency and damping ratio used by the pitch controller，as I could not find them in “Definition of the IEA Wind 22-Megawatt Offshore Reference Wind Turbine”. Thanks.

**Comment by gbarter** [comment]
A bit confused on the question here- You are asking about the 15-MW properties by referencing the 22-MW report?  Please see the [15-MW repository](https://github.com/IEAWindTask37/IEA-15-240-RWT) and floating-specific report.

**Comment by yangyangyang98** [comment]
Dear @gbarter 
Sorry, my expression was incorrect. What I want to ask is the relevant data of IEA 22MW FOWT. I could not find the platform-pitch natural frequency, as well as the natural frequency and damping ratio used by the pitch controller  in “Definition of the IEA Wind 22-Megawatt Offshore Reference Wind Turbine”. Thanks.

**Comment by dzalkind** [comment, marked as answer]
Hi @yangyangyang98,

[The natural frequency and damping ratio of the pitch controller can be found here](https://github.com/IEAWindTask37/IEA-22-280-RWT/blob/fe07471393c5b61eb0c7072bbfe2337ae80f3a81/OpenFAST/IEA-22-280-RWT-Semi/IEA-22-280-RWT-Semi.yaml#L8);; they are a function of the wind speed (`U_pc`).

The natural periods of the whole floating system, computed by RAFT, are as follows:
```
raft.surge_period,output,s,107.89755534840653,Surge natural period
raft.sway_period,output,s,108.43499424953963,Sway natural period
raft.heave_period,output,s,17.398511845041778,Heave natural period
raft.roll_period,output,s,27.281517618297688,Roll natural period
raft.pitch_period,output,s,27.292493450388193,Pitch natural period
raft.yaw_period,output,s,83.92729757773289,Yaw natural period
```

It was perhaps an oversight of mine to not include these in the report.  Should we host the WISDEM/WEIS outputs in this repo, @gbarter?

**Comment by gbarter** [comment]
Maybe we could add it to the [FAQ](https://github.com/IEAWindTask37/IEA-22-280-RWT/wiki/Frequently-Asked-Questions-(FAQ))?

**Reply by dzalkind** [reply]
Updated!

**Comment by yangyangyang98** [comment]
Thank you for your helpful answer！
